components/alibi-explain-server/Dockerfile

```diff
@@ -1,41 +1,65 @@
 ARG VERSION
-FROM seldonio/seldon-core-s2i-python37-ubi8:1.12.0-dev
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}:${VERSION} as base
+
+ARG VERSION
 LABEL name="Seldon Alibi Wrapper" \
       vendor="Seldon Technologies" \
-      version="1.12.0-dev" \
+      version="$VERSION" \
```

Comment:
Could you validate that this won't just be replaced by the release script? I think for several docker files this is what has basically led to the version being hardcoded